### DISCUSSION TITLE
Is there a page/directory of vendors selling Kura-based products?

### DISCUSSION THREAD

**Discussion by Humancell** [eclipse-kura/kura#4818] [Q&A] [answered]
Hello,

I know that Eurotech sold (or sells?) one or more products that were based on Kura.  Some of these appear to be very dated and based on older versions of Kura.

I was curious if there is a page or directory or list of companies that are selling turn-key products based on Kura?

Where could I find something like this?

Thank you!

Scott

**Comment by MMaiero** [comment, marked as answer]
Hi Scott,
thanks for reaching out!

At the following [link](https://iot.eclipse.org/adopters/) you'll find the list of adopters for Eclipse Kura along with other Eclipse IoT projects.
As you can see, [Eurotech](https://www.eurotech.com) is one of the Eclipse Kura adopters and the Eurotech lineup all uses [ESF](https://esf.eurotech.com), a commercially supported framework based on Eclipse Kura. 
I think you can reach out to the Eurotech Sales Representatives for your area, to get more details and feedback. 

Best regards,
Matteo

**Reply by Humancell** [reply]
Thank you for this information.  Looking at the link you provided, I only see Eurotech providing a commercially available physical device running ESF/Kura?

Also, working with a local company, the Eurotech box they have does not appear to be running nor upgradable to the latest versions of Kura.  I am struggling to find any alternatives that are upgradable to the latest versions.  We were hoping to not have to build our own one-off device.

We are running into all sorts of cellular connectivity issues with this older version.  The software just seems to be extremely temperamental and will only work after extensive tweaking and interaction.  Then it will start working, but then fails again after a reboot.  :-(

Thoughts?

**Reply by MMaiero** [reply]
I think that more details are needed and my suggestion is to reach out to a Eurotech Sales representative for your area to be able to go more into the details.
ESF releases are aligned with Eclipse Kura releases. As you can see from this page https://esf.eurotech.com/changelog/esf-730-release ESF 7.3.0 is based to Eclipse Kura 5.3.0 that is the latest Eclipse Kura release.
So you should be able to update your device to the latest version of Kura/ESF without the need to build your own version.

Best regards,
Matteo

**Reply by Humancell** [reply]
Thank you!

**Reply by Humancell** [reply]
We have reached out to Eurotech and they have indicated they will not, or are unable to, support modern roaming SIMs.

There are countless other projects that work just fine with these SIMs. This is why we are looking for alternate hardware sources for testing.

Thoughts?

**Reply by Humancell** [reply]
Our main objective is to debug this situation, since modern roaming SIMs are supported by any modern Linux solution and project using PPP and other protocols.

We can't believe there is anything specifically designed into the ESF or Kira projects to intentionally prevent proper operation.

Is this an incorrect assumption?
